src/LinuxTracing/TracerImpl.h

```diff
@@ -136,6 +136,8 @@ class TracerImpl : public Tracer {
   static constexpr uint64_t SAMPLING_RING_BUFFER_SIZE_KB = 16 * 1024;
   static constexpr uint64_t THREAD_NAMES_RING_BUFFER_SIZE_KB = 64;
   static constexpr uint64_t CONTEXT_SWITCHES_AND_THREAD_STATE_RING_BUFFER_SIZE_KB = 2 * 1024;
+  static constexpr uint64_t CONTEXT_SWITCHES_AND_THREAD_STATE_WITH_CALLSTACKS_RING_BUFFER_SIZE_KB =
```

Comment:
Shouldn't it be `WITH_STACKS` instead of `WITH_CALLSTACKS` as the stack dump is collected? Or do we already consider the frame pointer case and we want a name that kind of covers both?

Reply:
Let's use `WITH_STACKS`